src/protocol/sort/generate_permutation.rs

```diff
@@ -446,46 +438,4 @@ mod tests {
             perms_and_randoms[1].randoms_for_shuffle.1
         );
     }
-
-    #[tokio::test]
-    pub async fn malicious_sort_in_semi_honest() {
```

Comment:
Why is this test just gone?

Reply:
Since we changed the function output of convert_all_bits, we would need to take extra effort to maintain this test. This test is going to come back in PR with malicious sort.

Reply:
OK